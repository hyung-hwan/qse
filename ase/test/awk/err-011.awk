
function abc (x)
{
	local x;

	x = 20;
	print x;
	abc ();
}

BEGIN {
	abc ();
}
