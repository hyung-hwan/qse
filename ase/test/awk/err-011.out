PARSE ERROR: CODE [58] LINE [4] parameter 'x' redefined
